### PR TITLE
OPENNLP-1433 - Document how to do model releases

### PR DESCRIPTION
**Pull request by rzo1**
First draft of the model release documentation. Happy to get some feedback regarding wording / grammer / content, etc.

### CODE REVIEW

File: src/main/jbake/content/release-model.ad [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+
+- Elect a release manager.
+- Create a ticket for a new model release in JIRA. If you do not have permission to do so just ask to be given permissions on the mailing list.
+
+## Steps for the Release Manager
+
+The following steps need only to be performed once.
+
+- Make sure you have your PGP key entered into https://id.apache.org/.
+  Your KEYS will then be present in https://people.apache.org/keys/group/opennlp.asc.
```
Comment (kinow) [inline src/main/jbake/content/release-model.ad]: Hmm, that's 404 for me right now, https://people.apache.org/keys/group/opennlp.asc

File: src/main/jbake/content/release-model.ad [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+- Make sure you have your PGP keys password.
```
Comment (kinow) [inline src/main/jbake/content/release-model.ad]: key's?

File: src/main/jbake/content/release-model.ad [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+[ ] +1 Release the models as Apache OpenNLP Models [MODEL_VERSION]
+[ ] +0 meh, don't care
+[ ] -1 Do not release the models because of ${showstopper}
+
+Thanks!
+----
+
+## After a Successful Vote
+
+The vote is successful if at least 3 +1 votes are received from OpenNLP PMC members after a minimum of 72 hours of sending the vote email.
```
Comment (kinow) [inline src/main/jbake/content/release-model.ad:150]: Is a `-1` vote from a PMC a veto to the vote? If so might be worth mentioning here too?

File: src/main/jbake/content/release-model.ad [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+:category: news
+:idprefix:
+
+The Apache OpenNLP library is a machine learning based toolkit for the processing of natural language text.
+
+The Apache OpenNLP team is pleased to announce the release of <MODEL_TYPE>-<MODEL_VERSION> for Apache OpenNLP <OPENNLP_VERSION>.
+The models can <ADD_SUMMARY_HERE>
+
+Apache OpenNLP model and reports are available for download from our model download page:
+http://opennlp.apache.org/models.html
```
Comment (kinow) [inline src/main/jbake/content/release-model.ad]: https? (to save a redirect I think)

File: src/main/jbake/content/release-model.ad [hunk 5]
```diff
@@ -0,0 +1,255 @@
...
+#### Commit Website Changes.
+
+- Commit the website changes.
+- Rebuild opennlp-site and wait for site redeploy
+- Test and review the website. Test that all download links are working.
+
+### Post-Release Steps
+
+- Log the new version at https://reporter.apache.org/addrelease.html?opennlp.
+- Announce the new models on the OpenNLP Twitter.
```
Comment (kinow) [inline src/main/jbake/content/release-model.ad:216]: Worth mentioning that the password is in the SVN area, maybe with a link? (I **think** it was in SVN... somewhere... ?)

**Review by kinow — COMMENTED** [review]
Great stuff! A few comments added in the review. Thanks!

File: src/main/jbake/content/release-model.ad (hunk 1 above)
Comment (rzo1) [reply src/main/jbake/content/release-model.ad]: Will check. That is a copy of release.html (so should also be fixed there)

File: src/main/jbake/content/release-model.ad (hunk 3 above)
Comment (rzo1) [reply src/main/jbake/content/release-model.ad:150]: Releases may not be vetoed (see https://www.apache.org/foundation/voting.html), but normally a PMC -1 leads to a cancelled vote :-)

File: src/main/jbake/content/release-model.ad (hunk 5 above)
Comment (rzo1) [reply src/main/jbake/content/release-model.ad:216]: No Idea about the PW. If you know sth about it, I will adjust (also for release.html)

File: src/main/jbake/content/release-model.ad (hunk 1 above)
Comment (rzo1) [reply src/main/jbake/content/release-model.ad]: Keys are only managed via SVN / CDN via dist/release nowadays, see https://people.apache.org/keys/ - see Issue [OPENNLP-1438](https://issues.apache.org/jira/browse/OPENNLP-1438) for updating `release.ad`.

File: src/main/jbake/content/release-model.ad (hunk 5 above)
Comment (rzo1) [reply src/main/jbake/content/release-model.ad:216]: @jzonthemtn Do you know sth about the credentials? ;-) Perhaps available in the PMC SVN repo?

File: src/main/jbake/content/release-model.ad (hunk 4 above)
Comment (rzo1) [reply src/main/jbake/content/release-model.ad]: done

File: src/main/jbake/content/release-model.ad (hunk 5 above)
Comment (kinow) [reply src/main/jbake/content/release-model.ad:216]: Hm, I got confused with another ASF project that has the pwd there. But I couldn't find the OpenNLP one. I remember seeing it once, someone shared the location on Slack via a DM I think. But that was long ago and the channel doesn't have any trace of the info :disappointed_relieved:

**Review by kinow — APPROVED** [review]
:+1:

File: src/main/jbake/content/release-model.ad (hunk 5 above)
Comment (jzonthemtn) [reply src/main/jbake/content/release-model.ad:216]: No, I don't. What are the credentials needed for?

**Comment by jzonthemtn** [comment]
@rzo1 I think it captures everything. Thanks for writing this up. We can always update it as we go if we find things along the way.

**Review by jzonthemtn — APPROVED** [review]

**Comment by jzonthemtn** [comment]
Also, IIRC, when we did the release of those models it was a bit of a gray area because they didn't really fit into the ASF release process. ASF releases/votes are done on source code with convenience binary packages. What are models? From what we could find at the time it seemed like this problem was unique to OpenNLP. Maybe it is not anymore, but at some point it might be worthwhile having a larger conversation about releasing ML models for an ASF project.

File: src/main/jbake/content/release-model.ad (hunk 5 above)
Comment (kinow) [reply src/main/jbake/content/release-model.ad:216]: To log in to https://twitter.com/apacheopennlp

**Comment by rzo1** [comment]
@jzonthemtn I agree. Perhaps it is something we could address via the usual board minutes? So the ASF board is aware of it and maybe have a solution / way for it?

**Comment by jzonthemtn** [comment]
That's a great idea. I will go ahead and put it in the draft so it is there for the next submission.

**Review by mawiesne — APPROVED** [review]

File: src/main/jbake/content/release-model.ad (hunk 5 above)
Comment (jzonthemtn) [reply src/main/jbake/content/release-model.ad:216]: Ah! No, it isn't stored anywhere that I know of. 

@rzo1 You mentioned a PMC SVN repo. I did not know that is a thing. Is that a suitable place to store it?

Comment (rzo1) [reply src/main/jbake/content/release-model.ad:216]: We have specific SVN areas in TomEE due to (legacy) legal restriction on the TCK + a private SVN area afaik. But I wouldn't store them in VCS, I guess.

**Pull request merged**